rampwf/utils/submission.py

```diff
@@ -337,12 +338,12 @@ def bag_submissions(problem, cv, y_train, y_test, predictions_valid_list,
         test_idx = ([valid_is for (train_is, valid_is) in cv]
                     if step == 'valid' else None)
         score_dict = {}
-        for score in score_type:
+        for score_type in score_types:
             pred, scores = get_score_cv_bags(
-                score, pred_list, gt_list, test_is_list=test_idx
+                score_type, pred_list, gt_list, test_is_list=test_idx
             )
-            score_dict[score.name] = {key: val
-                                      for key, val in enumerate(scores)}
+            score_dict[score_type.name] = {
+                key: val for key, val in enumerate(scores)}
```

Comment:
The yapf or black style will be then

```python
score_dict[score_type.name] = {
                key: val
                for key, val in enumerate(scores)
}
```

In this case we can specify what the key in val instead of the general naming.